olp-cpp-sdk-dataservice-read/src/VersionedLayerClientImpl.cpp

```diff
@@ -519,6 +519,18 @@ bool VersionedLayerClientImpl::RemoveFromCache(const geo::TileKey& tile) {
   return RemoveFromCache(partition_id);
```

Comment:
After our switch to quad tree index this is also wrong and will probably never delete anything. Please fix!

Reply:
Will be fixed in separate PR